scania-digital-design-system/tegel
# fix(dropdown-filter): disabled can be opened

**Describe pull-request**  
This PR disables the `<tds-dropdown>` from being opened when disabled. The bug found was that it could be opened by pressing the chevron icon for the `filter` version even if it was `disabled`.

Also added some basic test for regressions on this functionality.


**Solving issue**  
#486 

**How to test**  
1. Go to Dropdown
2. Disable it and set it to filter
3. Try opening it by pressing the icon
4. It should not open.
5. Check out branch
6. Run all tests
7. Make sure they are valid and pass

**Suggested test steps**

**Screenshots**  
_If applicable, add screenshots to help explain_

**Additional context**  
_Add any other context about the pull-request here._
